git_hooks/pre-push

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
```

Comment:
Can this script please include a header in the same format we're adding into other [scripts](https://github.com/ARMmbed/mbedtls/blob/development/scripts/output_env.sh)?
e.g..
```
# <script name>.sh
#
# This file is part of mbed TLS (https://tls.mbed.org)
#
# Copyright (c) 2017, ARM Limited, All Rights Reserved
#
# Purpose
#
# To perform some action.
#
```